src/components/figure/figure.scss

```diff
@@ -0,0 +1,34 @@
+@use "../../design-tokens/colors.yml";
+@use "../../mixins/ms";
+
+/**
+ * Removes default margin from <figure>
+ */
+.c-figure {
+  margin: 0;
+}
+
+/**
+ * Contain and center image
+ */
+.c-figure__object {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 100%;
+}
+
+/**
+ * Styles the caption
+ */
+.c-figure__caption {
+  color: colors.$gray-dark;
```

Comment:
Instead of setting a hard `color` value here _and_ for `.t-dark`, maybe we could give the caption an opacity?